python/mxnet/gluon/block.py

```diff
@@ -1178,6 +1183,28 @@ def hybrid_forward(self, F, x, *args, **kwargs):
         # pylint: disable= invalid-name
         raise NotImplementedError
 
+    def optimize_for(self, backend, **kwargs):
```

Comment:
we're getting rid of this function right?

Reply:
Yes. Removed.